yammbs/torsion/_minimize.py

```diff
@@ -127,9 +134,11 @@ def _minimize_constrained(
     except (KeyError, AssertionError):
         pass
 
+    LOGGER.debug(f"Creating molecule, with conformer, from {input.mapped_smiles=}")
     molecule = Molecule.from_mapped_smiles(input.mapped_smiles, allow_undefined_stereo=True)
     molecule.add_conformer(Quantity(input.coordinates, "angstrom"))
```

Comment:
I could see the coordinates themselves being interesting for a super-involved debugging session, but it would also produce a ton of output.